conftest.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
```

Comment:
If this is meant to be a plugin installed by others, you cannot use a `conftest.py` file for this, you make this a normal module and use [setuptools entry points to register it](https://docs.pytest.org/en/stable/how-to/writing_plugins.html#making-your-plugin-installable-by-others).

Reply:
Thanks for the correction! I changed it and it's working now. I installed the package locally and tested in a fake project to make sure.